src/hooks/ApiHelperHook/useMarvelData.js

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Config from "react-native-config";
+
+function useMarvelData(parameter, apiType) {
+  const [data, setData] = useState([]);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const publicKey = Config.PUBLIC_KEY;
+  const hash = Config.HASH;
+  const ts = Config.T;
```

Comment:
Server tarafı için kullanılacak versiyonu kullanmışsınız. Client side versiyonu kullanacaktınız.